pkgs/applications/misc/obsidian/default.nix

```diff
@@ -12,20 +12,20 @@
 let
   inherit (stdenv.hostPlatform) system;
   pname = "obsidian";
-  version = "1.0.3";
+  version = "1.1.8";
   appname = "Obsidian";
   meta = with lib; {
-    description = "A powerful knowledge base that works on top of a local folder of plain text Markdown files";
+    description = "A powerful and extensible knowledge base that works on top of your local folder of plain text files.";
     homepage = "https://obsidian.md";
     downloadPage = "https://github.com/obsidianmd/obsidian-releases/releases";
     license = licenses.obsidian;
-    maintainers = with maintainers; [ atila conradmearns zaninime opeik ];
+    maintainers = with maintainers; [ atila conradmearns zaninime opeik berryp ];
   };
 
   filename = if stdenv.isDarwin then "Obsidian-${version}-universal.dmg" else "obsidian-${version}.tar.gz";
   src = fetchurl {
     url = "https://github.com/obsidianmd/obsidian-releases/releases/download/v${version}/${filename}";
-    sha256 = if stdenv.isDarwin then "sha256-DYF9fEpZaP4tD/eeZAegDahR7UZyroqNB9bn2U7sgXs=" else "sha256-MpQk5g4184ZkCAjLU5Ug0ReWgVADskS1QuMcnPdNofs=";
+    sha256 = if stdenv.isDarwin then "047cqlyc7s3xf7f5va7gf6ay898lrx34jn06w36a76004qj33bvs" else "1p8l9hpcarlg7ky4p75nd2cbn9rdfs3v2fgd1bkh4gs0vnsryr0h";
```

Comment:
Any reason for changing the hashes to the old-style ones? You might want to convert them to the newer SRI format in case the tool you used only produces the old ones: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-hash-to-sri.html

Reply:
Thanks for the link to docs!